src/utils/notifications.ts

```diff
@@ -118,10 +120,9 @@ export const showSyncIgnoredRepo = (repoPath: string, parentRepoPath: string) =>
 		});
 };
 
-export const showFreeTierLimitReached = (repoPath: string, isNewPrivateRepo=false) => {
+export const showFreeTierLimitReached = (repoPath: string, isNewPrivateRepo=false, canAvailTrial: IUserSubscriptionInfo) => {
```

Comment:
I think type of `canAvailTrial` should be boolean instead of `IUserSubscriptionInfo`